boards/nucleo_f429zi/src/virtual_uart_rx_test.rs

```diff
@@ -0,0 +1,87 @@
+//! Test reception on the virtualized UART by creating two readers that
```

Comment:
I imagine we don't need a copy of this?

Reply:
Oh, yes. This is not architecture specific. Removed.